src/app/mat-select-search/mat-select-search.component.scss

```diff
@@ -116,3 +116,7 @@ $mat-option-height: 3em;
   padding-left: 16px;
   padding-bottom: 2px;
 }
+
+::ng-deep .mat-tooltip.tooltip {
+  font-size: 0.8em;
```

Comment:
why is this needed? the user could adapt the font size himself

Reply:
hum yes maybe i need to test it to update the css directly in the exemple and not in the component